### ISSUE TITLE
Remove search boxes in schema explorer panel, replace with a single search box at bucket level

### ISSUE THREAD

**Issue by alexpaxton** [influxdata/chronograf#4629]
Must be completed after #4625 

- [ ] One search box within each bucket
- [ ] Should filter entire contents of bucket
- [ ] Highlight matching text in results
- [ ] all items are closed, it should still search through everything and show the open ones that it finds

![filtered_schema_explorer](https://user-images.githubusercontent.com/11937365/47323912-a6250700-d612-11e8-8762-a4ce73812a37.jpg)

**Comment by chnn** [comment]
Closing in favor of updated #4625
